tutorials/holoscan_debugging/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as base
+
+RUN apt update && \
+    apt install -y gdb
```

Comment:
I tested `debug_legacy.sh` on my x86 machine, the `endoscopy_tool_tracking` app build process reports error - missing `ffmpeg`. 

After adding `ffmpeg` to `apt install -y`, all builds and runs without issue. Could you double check?

Reply:
Interesting, I'll look into this issue. @zhijinl could you please confirm whether you started from a clean build directory? If not, you can clear out your previous build with:
```
rm -rf build/endoscopy_tool_tracking
```

Reply:
Oh yes, you are correct - I wasn't building in a clean directory. I just tested a clean build, all works fine! Let's resolve this then

Reply:
Added a sentence about clearing the build directory in the Prerequisites section, thanks for catching that!